.github/helper/translation.py

```diff
@@ -4,7 +4,7 @@
 errors_encounter = 0
 pattern = re.compile(r"_\(([\"']{,3})(?P<message>((?!\1).)*)\1(\s*,\s*context\s*=\s*([\"'])(?P<py_context>((?!\5).)*)\5)*(\s*,\s*(.)*?\s*(,\s*([\"'])(?P<js_context>((?!\11).)*)\11)*)*\)")
 words_pattern = re.compile(r"_{1,2}\([\"'`]{1,3}.*?[a-zA-Z]")
-start_pattern = re.compile(r"_{1,2}\([f\"'`]{1,3}")
+start_pattern = re.compile(r"_{1,2}\([\"'`]{1,3}")
```

Comment:
why? 

That `f` was intentional so that we can throw error stating `f-strings` are not valid in translation syntax.

Reply:
Copied that from https://github.com/frappe/frappe/pull/11689/files#diff-0845f0e1c9483ed231756c5a9cc0025e56e12fea99eaf33dcc38aaf107b59b70 .
Should we introduce the `f` in frappe as well?